From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Wed, 5 Oct 2016 11:14:39 +0200
Subject: use changed function name after gsoap 2.8.35

Upstream of gSOAP change the function soap_new_block to soap_alloc_block
in stdsoap2.h. By this we have to adopt the call in
provider/libserver/ECConversion.cpp line 274.
---
 provider/libserver/ECConversion.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/provider/libserver/ECConversion.cpp b/provider/libserver/ECConversion.cpp
index 463754f..4630637 100644
--- a/provider/libserver/ECConversion.cpp
+++ b/provider/libserver/ECConversion.cpp
@@ -270,7 +270,7 @@ SOAP_FMAC3 struct entryList52X * SOAP_FMAC4 soap_in_entryList52X(struct soap *so
 		{	soap->error = SOAP_TAG_MISMATCH;
 			if (soap_flag___ptr && soap->error == SOAP_TAG_MISMATCH)
 			{	unsigned int *p;
-				soap_new_block(soap);
+				soap_alloc_block(soap);
 				for (a->__size = 0; !soap_element_begin_in(soap, "item", 1, type); ++a->__size) {
 					p = (unsigned int *)soap_push_block(soap, NULL, sizeof(unsigned int));
 					soap_default_unsignedInt(soap, p);
